alphadia/quadrupole.py

```diff
@@ -13,7 +6,7 @@
 from alphadia import utils
 
 
-@alphatims.utils.njit
```

Comment:
Did you check the alphatims function? I wonder why I did this in the first place ( I already see some weird numbs thing happening :D )

Reply:
this is the `@alphatims.utils.njit` decorator
```
def njit(_func=None, *args, **kwargs):
    import numba
    return numba.njit(_func, *args, **kwargs)
```

Reply:
So much value